src/Plugin/views/relationship/CiviCrmBridgeRelationshipBase.php

```diff
@@ -52,10 +53,17 @@ public function query() {
     $first_alias = $this->query->addTable($this->definition['table'], $this->relationship, $first_join);
 
     // Relate the first join to the base table defined.
+    // If the table is _not_ a CiviCRM table, assume it is for the default
+    // connection, and provide the fully qualified table name so that we can
+    // query across the databases.
+    $table = $this->definition['base'];
+    if (strpos($table, 'civicrm_') !== 0) {
+      $table = Database::getConnection()->getFullQualifiedTableName($table);
+    }
```

Comment:
@KarinG this should fix errors on `user__field_user_team` not working correctly.

Or. Shoot. It might now. Because Views has more problems. You're having errors on _fields_ not relationships.

Reply:
So if I checkout:
root@544a28c2257f:/var/www/drupal/web/sites/default/modules/civicrm_entity# git checkout f2bc70d719431bf8f756164d9bb55570f61b14a3

All is well 
![image](https://user-images.githubusercontent.com/5340555/125015099-0f9c0c80-e02c-11eb-94cd-60377a77e854.png)

But if I 
git checkout 71635c6a4ff3d424155f432c63a4e73df65b74ab - Add initial Views testing 
![image](https://user-images.githubusercontent.com/5340555/125015262-61449700-e02c-11eb-9293-dcf2a3b98617.png)

Something in the initial Views testing PRs stopped this two databases site view from working -> 
![image](https://user-images.githubusercontent.com/5340555/125015399-ad8fd700-e02c-11eb-9c20-6ae5bfba546a.png)

Reply:
So there is a Relationship -> (User) field_user_team: Taxonomy term

Reply:
I just checked out 8.x-3.x again and applied this PR -> unfortunately still the same error -> 

Drupal\Core\Database\DatabaseExceptionWrapper: Exception in CLIN Calendar[clin_calendar]: SQLSTATE[42S02]: Base table or view not found: **1146 Table 'civicrm_sace.user__field_user_team' doesn't exist:** SELECT civicrm_activity.id AS id, civicrm_contact_civicrm_activity_contact.id AS civicrm_contact_civicrm_activity_contact_id, civicrm_contact_civicrm_activity_contact_1.id AS civicrm_contact_civicrm_activity_contact_1_id, civicrm_contact_civicrm_activity_contact_2.id AS civicrm_contact_civicrm_activity_contact_2_id, users_field_data_civicrm_uf_match.uid AS users_field_data_civicrm_uf_match_uid, taxonomy_term_field_data_user__field_user_team.tid AS taxonomy_term_field_data_user__field_user_team_tid FROM {civicrm_activity} civicrm_activity LEFT JOIN {civicrm_activity_contact} civicrm_activity_contact ON civicrm_activity.id = civicrm_activity_contact.activity_id AND civicrm_activity_contact.record_type_id = :views_join_condition_0 LEFT JOIN {civicrm_contact} civicrm_contact_civicrm_activity_contact ON civicrm_activity_contact.contact_id = civicrm_contact_civicrm_activity_contact.id LEFT JOIN {civicrm_activity_contact} civicrm_activity_contact2 ON civicrm_activity.id = civicrm_activity_contact2.activity_id AND civicrm_activity_contact2.record_type_id = :views_join_condition_1 LEFT JOIN {civicrm_contact} civicrm_contact_civicrm_activity_contact_1 ON civicrm_activity_contact2.contact_id = civicrm_contact_civicrm_activity_contact_1.id LEFT JOIN {civicrm_activity_contact} civicrm_activity_contact3 ON civicrm_activity.id = civicrm_activity_contact3.activity_id AND civicrm_activity_contact3.record_type_id = :views_join_condition_2 LEFT JOIN {civicrm_contact} civicrm_contact_civicrm_activity_contact_2 ON civicrm_activity_contact3.contact_id = civicrm_contact_civicrm_activity_contact_2.id LEFT JOIN {civicrm_uf_match} civicrm_contact_civicrm_activity_contact__civicrm_uf_match ON civicrm_contact_civicrm_activity_contact.id = civicrm_contact_civicrm_activity_contact__civicrm_uf_match.contact_id LEFT JOIN drupal_sace.users_field_data users_field_data_civicrm_uf_match ON civicrm_contact_civicrm_activity_contact__civicrm_uf_match.uf_id = users_field_data_civicrm_uf_match.uid LEFT JOIN {user__field_user_team} users_field_data_civicrm_uf_match__user__field_user_team ON users_field_data_civicrm_uf_match.uid = users_field_data_civicrm_uf_match__user__field_user_team.entity_id AND users_field_data_civicrm_uf_match__user__field_user_team.deleted = :views_join_condition_3 LEFT JOIN {taxonomy_term_field_data} taxonomy_term_field_data_user__field_user_team ON users_field_data_civicrm_uf_match__user__field_user_team.field_user_team_target_id = taxonomy_term_field_data_user__field_user_team.tid WHERE (((civicrm_activity.activity_type_id IN(:civicrm_activity_activity_type_id__0, :civicrm_activity_activity_type_id__1, :civicrm_activity_activity_type_id__2, :civicrm_activity_activity_type_id__3, :civicrm_activity_activity_type_id__4, :civicrm_activity_activity_type_id__5, :civicrm_activity_activity_type_id__6, :civicrm_activity_activity_type_id__7, :civicrm_activity_activity_type_id__8, :civicrm_activity_activity_type_id__9, :civicrm_activity_activity_type_id__10, :civicrm_activity_activity_type_id__11, :civicrm_activity_activity_type_id__12, :civicrm_activity_activity_type_id__13, :civicrm_activity_activity_type_id__14, :civicrm_activity_activity_type_id__15, :civicrm_activity_activity_type_id__16))) AND ((taxonomy_term_field_data_user__field_user_team.tid = :taxonomy_term_field_data_tid))) AND ((DATE_FORMAT((civicrm_activity.activity_date_time + INTERVAL -21600 SECOND), '%Y-%m-%d\T%H:%i:%s') >= DATE_FORMAT(('2021-06-09T02:43:40' + INTERVAL -21600 SECOND), '%Y-%m-%d\T%H:%i:%s'))); Array ( [:taxonomy_term_field_data_tid] => 13 [:views_join_condition_0] => 1 [:views_join_condition_1] => 3 [:views_join_condition_2] => 2 [:views_join_condition_3] => 0 [:civicrm_activity_activity_type_id__0] => 66 [:civicrm_activity_activity_type_id__1] => 65 [:civicrm_activity_activity_type_id__2] => 67 [:civicrm_activity_activity_type_id__3] => 68 [:civicrm_activity_activity_type_id__4] => 71 [:civicrm_activity_activity_type_id__5] => 75 [:civicrm_activity_activity_type_id__6] => 76 [:civicrm_activity_activity_type_id__7] => 77 [:civicrm_activity_activity_type_id__8] => 80 [:civicrm_activity_activity_type_id__9] => 79 [:civicrm_activity_activity_type_id__10] => 69 [:civicrm_activity_activity_type_id__11] => 70 [:civicrm_activity_activity_type_id__12] => 78 [:civicrm_activity_activity_type_id__13] => 82 [:civicrm_activity_activity_type_id__14] => 81 [:civicrm_activity_activity_type_id__15] => 84 [:civicrm_activity_activity_type_id__16] => 83 ) in Drupal\views\Plugin\views\query\Sql->execute() (line 1543 of /var/www/drupal/web/core/modules/views/src/Plugin/views/query/Sql.php).